.github/workflows/release-please.yml

```diff
@@ -0,0 +1,38 @@
+name: Release Please
+on:
+  push:
+    branches:
+      - main
+
+jobs:
+  release-please:
+    runs-on: ubuntu-latest
+    steps:
+      - uses: googleapis/release-please-action@v4
+        id: release
+        with:
+          config-file: release-please-config.json
+          manifest-file: .release-please-manifest.json
```

Comment:
We potentially need to specify a token here - I can't remember the specifics but I have a recollection about the default token not having the right permissions.

Reply:
Done